Cargo.toml

```diff
@@ -1,11 +1,5 @@
 [workspace]
-members = [
-    "guests",
-    "host",
-    "lib",
-    "primitives",
-    "testing/ef-tests"
```

Comment:
Removed because the standard Ethereum tests fail? Maybe we should add something similar again to `is_taiko_block` so these tests on standard Ethereum blocks can still work with our taiko revm.

Reply:
Oh, yeah, some tests and dependency are break, so i will add back